File: 0-AdVeYaşYazdırmak/AdVeYasYazdirmak.java
import java.util.*;
public class AdVeYasYazdirmak {
 public static void main(String arg[]) {

  Scanner input = new Scanner(System.in);

  // “isim”in string olarak tanımlanması
  String isim;
  
  // “yil”, “yas”, “buyil”ın tanımlanması
  int yil, yas, buyil = 2017; 
  System.out.print("Isminizi giriniz: ");
  
  // “isim” ve “yil”ın girilmesi
  isim = input.nextLine(); 
  System.out.print("Dogum yilinizi giriniz: ");
  yil = input.nextInt();

  // “yas”ın değerinin hesaplanması
  yas = buyil - yil;

  // “isim” ve “yas”ın ekrana yazdırılması
  System.out.println("Isminiz: " + isim); 
  System.out.println("Yasiniz: " + yas);
  
  input.close();
 }
}

File: 0-AdVeYaşYazdırmak/AdVeYasYazdirmak.cpp
#include <iostream>

using namespace std;
int main()
{
    // “isim”in string olarak tanımlanması
    string isim;
  
    // “yil”, “yas”, “buyil”ın tanımlanması
    int yil, yas, buyil = 2019; 

    // “isim” ve “yil”ın girilmesi
    cout << "Isminizi giriniz: ";
    cin >> isim;
    cout << "Dogum yilinizi giriniz: ";
    cin >> yil;

    // “yas”ın değerinin hesaplanması
    yas = buyil - yil;

    // “isim” ve “yas”ın ekrana yazdırılması
    cout << "Isminiz: " << isim <<  "Yasiniz: " << yas ; 
    
    return 0;
}
